src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -350,6 +351,13 @@ object Task {
 
   }
 
+  /** @return a stream of task local volumes, extrapolating them from the app spec */
+  def localVolumes(app: AppDefinition): Iterable[LocalVolume] = {
+    app.container.toSet[Container].flatMap(VolumesModule.localVolumes.collect).map{ volume =>
```

Comment:
formatting nit-pick: We use a space between map (or another method) and a block. E.g. `.map {` instead of `.map{`.

A better place for this method would be in the companion object of `LocalVolume`. Or maybe even in the `VolumesModule`.